tests/init/test_init.py

```diff
@@ -253,7 +253,7 @@ def test_init_minimal_interaction_required_yes_git_no_network(
     paths = {p.relative_to(created_dir) for p in created_dir.iterdir()}
     assert paths == {Path(".git"), Path("myapp")}
     git_rev_list = subprocess.run(
-        ["git", "rev-list", "--max-parents=0", "HEAD"], cwd=created_dir, capture_output=True, text=True
+        ["git", "rev-list", "--max-parents=0", "HEAD"], cwd=created_dir, capture_output=True, text=True, check=True
```

Comment:
similarly, the `returncode` here is being asserted below, which is prefereable to raising an exception here (ie causes a test failure rather than a test error)